activesupport/lib/active_support/rails.rb

```diff
@@ -1,17 +1,15 @@
-# This is private interface.
```

Comment:
Ah it's still expressed below. Curious why we'd want a warning up top and then not keep the whole privacy bit there as well.